bridge-www/src/App.tsx

```diff
@@ -75,11 +75,11 @@ function App() {
       To : <input type="text" value={ncAddress} onChange={event => { setNcAddress(event.target.value) }}/>
       <br/>
       {
-        contract === null || account === null || amount === null || isNaN(parseInt(amount)) || !isAddress(ncAddress)
+        contract === null || account === null || amount === null || (parseFloat(amount) * 100).toString().indexOf(".") !== -1 || !isAddress(ncAddress)
```

Comment:
```suggestion
        contract === null || account === null || amount === null || parseFloat(amount).toFixed(2) === amount || !isAddress(ncAddress)
```

Reply:
Thanks for your suggestion but it will not work as expected. let's think when `amount` variable is `"0"`:

```javascript
const amount = "0";
parseFloat(amount).toFixed(2) === amount; // "0.00" === "0" (false)
```